infra/Makefile.toml

```diff
@@ -5,18 +5,36 @@ extend = [
   { path = "scripts/ethapi.toml" },
   { path = "scripts/node.toml" },
   { path = "scripts/testnet.toml" },
-  { path = "scripts/testnode.toml" }
+  { path = "scripts/testnode.toml" },
+  { path = "scripts/subnet.toml" }
 ]
 
 [config]
 default_to_workspace = false
 
 [env]
-CHAIN_NAME = { value = "root", condition = { env_not_set = ["CHAIN_NAME"] }}
+# General network-specific parameters
+SUBNET_ID = { value = "root", condition = { env_not_set = ["SUBNET_ID"] }}
 BALANCE = { value = "1000", condition = { env_not_set = ["BALANCE"] }}
 BASE_FEE = { value = "1000", condition = { env_not_set = ["BASE_FEE"] }}
 TIMESTAMP = { value = "1680101412", condition = { env_not_set = ["TIMESTAMP"] }}
-BASE_DIR="${HOME}/.ipc/${CHAIN_NAME}"
+POWER_SCALE = { value = "3", condition = { env_not_set = ["POWER_SCALE"] }}
+CMT_P2P_HOST_PORT = { value = "26656", condition = { env_not_set = ["CMT_P2P_HOST_PORT"] }}
+CMT_RPC_HOST_PORT = { value = "26657", condition = { env_not_set = ["CMT_RPC_HOST_PORT"] }}
+ETHAPI_HOST_PORT = { value = "8545", condition = { env_not_set = ["ETHAPI_HOST_PORT"] }}
+
+# IPC subnet related parameters
+# Use calibration as default value
+NODE_NAME = { value = "ipc-node", condition = { env_not_set = ["NODE_NAME"] } }
+PARENT_ENDPOINT = { value = "https://api.calibration.node.glif.io/rpc/v1", condition = { env_not_set = ["PARENT_ENDPOINT"] } }
+PARENT_GATEWAY = { value = "0x56948d2CFaa2EF355B8C08Ac925202db212146D1", condition = { env_not_set = ["PARENT_GATEWAY"] } }
+PARENT_REGISTRY = { value = "0x6A4884D2B6A597792dC68014D4B7C117cca5668e", condition = { env_not_set = ["PARENT_REGISTRY"] } }
+NETWORK_TYPE = { value = "test", condition = { env_not_set = ["NETWORK_TYPE"] } }
+VALIDATOR_PRIV_KEY = { value = "~/.ipc/priv.key", condition = { env_not_set = ["VALIDATOR_PRIV_KEY"] } }
+TOPDOWN_CHAIN_HEAD_DELAY = { value = "10", condition = { env_not_set = ["TOPDOWN_CHAIN_HEAD_DELAY"] } }
+
+# Deployment-related
+BASE_DIR="${HOME}/.ipc/${SUBNET_ID}"
```

Comment:
Just to confirm, the `SUBNET_ID` is something like `/r123/f410123...abc/f064`, and they will actually map to a nice nested directory structure, right?

Reply:
Correct, so if you have more than one subnet anchored to `/r123` they will be in `/r123/subnet1` and `/r123/subnet2` (I already tested this one just in case :) )